docs/notes-pa.md

```diff
@@ -3,4 +3,166 @@ how the permutation argument is formed.
 
 
 Permutation argument
-==============
\ No newline at end of file
+==============
+
+Within PLONK, the permutation 
+polynomials are used to compare 
+arguments about the proving 
+scheme.  
+
+These arguments are 
+checking that wire values, 
+within the circuit, are
+being correctly copied. 
+As is noted in other parts
+of the docs, there is the 
+need for a copy check in 
+in SNARK schemes. This is 
+crucial in our zk-SNARK
+scheme, to ensure the 
+correctness of wires when 
+bundling similar wires, 
+and for ensuring values are
+travelling the correct way
+along wires.
```

Comment:
What do you mean by "ensure values are traveling the correct way along wires"?

Reply:
I meant 'copy checks', I'll change it now.